byron/chain/executable-spec/src/Cardano/Spec/Chain/STS/Block.hs

```diff
@@ -94,18 +101,27 @@ bUpdPayload b = (b ^. bBody ^. bUpdProp, b ^. bBody ^. bUpdVotes)
 bEndorsment :: Block -> (ProtVer, VKey)
 bEndorsment b = (b ^. bBody ^. bProtVer, b ^. bHeader ^. bhIssuer)
 
-
 -- | Compute the size (in words) that a block takes.
```

Comment:
We should correct this comment as well. We are actually computing `the abstract size that a block takes`.

Reply:
done